python/ray/autoscaler/aws/defaults.yaml

```diff
@@ -104,6 +104,17 @@ cluster_synced_files: []
 # should sync to the worker node continuously
 file_mounts_sync_continuously: False
 
+# Patterns for files to exclude when running rsync up or rsync down
+rsync_exclude:
+    - "**/.git"
+    - "**/.git/**"
+
+# Pattern files to use for filtering out files when running rsync up or rsync down. The file is searched for
+# in the source directory and recursively through all subdirectories. For example, if .gitignore is provided
+# as a value, the behavior will match git's behavior for finding and using .gitignore files.
+rsync_filter:
+    - ".gitignore"
+
```

Comment:
should these be defaults? @richardliaw, @wuisawesome

Reply:
Since it's backwards incompatible to change `defaults.yaml`, I think we should stick with them empty by default. However, we can have them ignore git in the example-full files, which don't have backwards compatibility implications.